assets/locales/cs/genericComponents.json

```diff
@@ -103,7 +103,7 @@
   "InitialSyncInProgress": "Probíhá počáteční synchronizace...",
   "NodeSettings": "Nastavení Uzlu...",
   "RemoveNode": "Odstranit Uzel...",
-  "RemovingNode": "Odstraňování Uzlu...",
+  "Removing node...": "Odstraňování Uzlu...",
```

Comment:
do we want to keep this consistent with other keys using CamelCase? @jgresham

Reply:
Yeah, nice catch. Was testing this because the translation isn't being used properly after clicking Remove Node. All other translations on the modal work fine. Still not sure why it only shows "RemovingNode" and not using the translation